openfisca_aotearoa/variables/acts/citizenship/citizenship.py

```diff
@@ -116,8 +133,10 @@ def days_since_n_years_ago(day, n=1):
     """
     try:
         date_n_years_ago = day.replace(year=day.year - n)
+        
+        #print("date_n_years_ago", day, " subtract", n, "-->", date_n_years_ago)
         # The days in that rolling year could  be 365 or 366
-        days = (day - date_n_years_ago).days - 1  # subtract one to not include that day
+        days = (day - date_n_years_ago).days  # subtract one to not include that day
```

Comment:
should the "subtract one" comment still be here?

Reply:
:+1: removing